cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureReportBuilder.java

```diff
@@ -460,10 +456,9 @@ protected void addMeasureDescription(MeasureReportGroupComponent reportGroup, Me
         }
     }
 
-    protected void addExtensionImprovementNotation(
-            MeasureReportGroupComponent reportGroup, MeasureDef measureDef, GroupDef groupDef) {
+    protected void addExtensionImprovementNotation(MeasureReportGroupComponent reportGroup, GroupDef groupDef) {
         // if already set on Measure, don't set on groups too
-        if (!measureDef.useMeasureImpNotation()) {
+        if (groupDef.isGroupImprovementNotation()) {
```

Comment:
The previous if was a negative condition an this one is positive.  Just wanted to make sure that was your intention.

Reply:
good question, luckily these are looking at different references to check. One is Measure level the other group level. this is correct. behavior